### PR TITLE
squad_v2 and truthful_qa evaluation results

### PR DESCRIPTION
**Pull request by Addi-11**
- Onboarded squad_v2 dataset.
- Updated coherence scores for truthfulqa_generation datatset.

### CODE REVIEW

**Review by guptha23 — DISMISSED** [review]

File: assets/evaluation_results/squad_v2_databricks-dbrx-base_question_answering/spec.yaml [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+name: squad_v2_databricks-dbrx-base_question_answering
+version: 2.23.05
+display_name: databricks-dbrx-base_squad_v2_text_generation
+description: databricks-dbrx-base run for squad_v2 dataset
+dataset_family: squad_v2
+dataset_name: squad_v2
+
+model_name: databricks-dbrx-base
+model_version: "3"
+model_asset_id: azureml://registries/azureml-staging/models/databricks-dbrx-base/versions/3
```
Comment (iamrk04) [inline assets/evaluation_results/squad_v2_databricks-dbrx-base_question_answering/spec.yaml:11]: Is there any specific reason we have chosen the model from `azureml-staging` registry? Ideally it should be from one of the prod registries. Let's verify the `model_name`, `model_version` and `model_asset_id` for all the models.

**Review by iamrk04 — CHANGES_REQUESTED** [review]

**Review by guptha23 — DISMISSED** [review]

File: assets/evaluation_results/truthfulqa_generation_mistralai-mixtral-8x7b-v01_question_answering/spec.yaml [hunk 2]
```diff
@@ -1,28 +1,28 @@
...
+version: 2.23.05
+display_name: mistralai-Mixtral-8x7B-v01_truthfulqa_generation_text_generation
+description: mistralai-mixtral-8x7b-v01  for truthfulqa_generation dataset
 dataset_family: truthfulqa
 dataset_name: truthfulqa_generation
 
-model_name: mistralai-Mixtral-8x7B-v01
-model_version: "4"
-model_asset_id: azureml://registries/azureml/models/mistralai-Mixtral-8x7B-v01/versions/4
+model_name: mistralai-mixtral-8x7b-v01
```
Comment (iamrk04) [inline assets/evaluation_results/truthfulqa_generation_mistralai-mixtral-8x7b-v01_question_answering/spec.yaml:9]: looks like this update has been done by mistake?

File: assets/evaluation_results/truthfulqa_generation_mistralai-mixtral-8x7b-v01_question_answering/spec.yaml [hunk 3]
```diff
@@ -1,28 +1,28 @@
 type: evaluationresult
 name: truthfulqa_generation_mistralai-mixtral-8x7b-v01_question_answering
-version: 2.19.040
-display_name: truthfulqa_generation_mistralai-Mixtral-8x7B-v01_question_answering
-description: mistralai-Mixtral-8x7B-v01 run for truthfulqa_generation dataset
+version: 2.23.05
+display_name: mistralai-Mixtral-8x7B-v01_truthfulqa_generation_text_generation
+description: mistralai-mixtral-8x7b-v01  for truthfulqa_generation dataset
```
Comment (iamrk04) [inline assets/evaluation_results/truthfulqa_generation_mistralai-mixtral-8x7b-v01_question_answering/spec.yaml:5]: May I know why were the `display_name` and `description` changed? This shouldn't have happened. Please verify for all the assets.

**Review by iamrk04 — CHANGES_REQUESTED** [review]

File: assets/evaluation_results/truthfulqa_generation_mistralai-mixtral-8x7b-v01_question_answering/spec.yaml (hunk 2 above)
Comment (Addi-11) [reply assets/evaluation_results/truthfulqa_generation_mistralai-mixtral-8x7b-v01_question_answering/spec.yaml]: fixed

**Review by iamrk04 — APPROVED** [review]

**Pull request merged**